gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1-bin.zip
```

Comment:
is there a reason not to use 8.2.1?

Reply:
BTW, I think, that updating gradle often also involve update of wrapper.jar and gradlew files (updated when running f.e. `./gradlew wrapper --gradle-version=8.2.1`)

Reply:
> is there a reason not to use 8.2.1?

No, it's just me lagging behind the progress. ;)